### DISCUSSION TITLE
Weird mouse keyboard intereation with new update

### DISCUSSION THREAD

**Discussion by MarcioRF92** [redphx/better-xcloud#203] [General]
I'm feeling that the keyboard and mouse functionality has changed, after the 3.0 update, I went to test Halo multiplayer just now, and it seems stuck, as if I were emulating the controller's movements, it became unplayable.
Before this update it was working perfectly.

**Comment by trismeg93** [comment]
Same problem here! How can a function that worked so well be eliminated, worsening the overall quality after an update? Very sad...

**Comment by DocS788** [comment]
y same sensation like the _emulation of the controller_,, tried now in medieval dynasty..but i think other games so

**Comment by redphx** [comment]
I still don't understand what it is about

**Reply by MarcioRF92** [reply]
redphx, good morning! I wanted to give feedback, for some reason, the new version 3.0 has worsened the performance of the keyboard and mouse in xcloud, it seems that it is emulating the control. In the previous version, I think in the beta, the keyboard and mouse worked fluidly in Halo Infinity for example, when I went to test the new version this week, this feature got worse. Note that two other users reported the same thing.

**Reply by redphx** [reply]
Not a bug #208
